compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmDesugarPhase.java

```diff
@@ -237,6 +238,15 @@ private static void encodeFunctionIdentifiers(List<BIRFunction> functions,
                 parameter.name = Names.fromString(encodeNonFunctionIdentifier(parameter.name.value,
                                                                               encodedVsInitialIds));
             }
+            if (function.type.tsymbol != null) {
+                for (BVarSymbol parameter : ((BInvokableTypeSymbol) function.type.tsymbol).params) {
+                    if (parameter.name == null) {
+                        continue;
+                    }
```

Comment:
Can `parameter.name` be `null`? If it can't be shall we remove this in both the places?